action.yml

```diff
@@ -9,10 +9,22 @@ runs:
   steps:
     - name: Checkout
       uses: actions/checkout@v3
+    - name: Determine Project Type
+      id: project_type
+      run: |
+        project_type="${{ inputs.project-type }}"
```

Comment:
question: if you have the `inputs.project-type` available here, won't you have it available in the `Download Detekt Configuration` step also? Is this added step necessary at all?

Reply:
Actually this step might not be needed. I just did not delete it. Good point.